controller/volume_controller.go

```diff
@@ -2449,8 +2455,8 @@ func (vc *VolumeController) getIsSchedulableToDiskNodes(v *longhorn.Volume, node
 		if err != nil {
 			continue
 		}
-		for fsid, diskStatus := range node.Status.DiskStatus {
-			diskSpec, exists := node.Spec.Disks[fsid]
+		for diskID, diskStatus := range node.Status.DiskStatus {
```

Comment:
This looks weird... IIUC, the key of the disk map in node spec/status should be disk name rather than fsid/diskID/diskUUID

Reply:
Interesting... Yeah, it should be
```
for diskName, diskStatus := range node.Status.DiskStatus {
```
rather than
```
for fsid, diskStatus := range node.Status.DiskStatus {
```
 The existing codes is using the wrong name.

Will fix it in this PR.